package.json

```diff
@@ -71,43 +74,40 @@
     }
   },
   "dependencies": {
-    "@angular/animations": "11.0.5",
-    "@angular/common": "11.0.5",
-    "@angular/core": "11.0.5",
-    "@angular/forms": "11.0.5",
-    "@angular/platform-browser": "11.0.5",
-    "@angular/platform-browser-dynamic": "11.0.5",
-    "@angular/router": "11.0.5",
+    "@angular/animations": "11.2.4",
+    "@angular/common": "11.2.4",
+    "@angular/core": "11.2.4",
+    "@angular/forms": "11.2.4",
+    "@angular/platform-browser": "11.2.4",
+    "@angular/platform-browser-dynamic": "11.2.4",
+    "@angular/router": "11.2.4",
     "@netbasal/content-loader": "^1.1.0",
     "faker": "^4.1.0",
-    "helpful-decorators": "^2.0.6",
-    "lodash": "4.17.20",
-    "rxjs": "6.5.5",
-    "schematics-utilities": "2.0.2",
```

Comment:
@NetanelBasal here, the `schematics-utilities` to build the libraries were different to the package version.